config/disks.go

```diff
@@ -14,30 +14,30 @@ const (
 
 // applied policy enum
 const (
-	DISKS_POLICY_INCLUDE = "explicit_include_policy"
-	DISKS_POLICY_INCLUDE_BY_REGEX = "explicit_include_by_regex_policy"
-	DISKS_POLICY_EXCLUDE = "explicit_exclude_policy"
-	DISKS_POLICY_EXCLUDE_BY_REGEX = "explicit_exclude_by_regex_policy"
-	DISKS_POLICY_CONFLICTING = "unallowed_definition_in_include_and_exclude_policy"
+	DISKS_POLICY_INCLUDE              = "explicit_include_policy"
+	DISKS_POLICY_INCLUDE_BY_REGEX     = "explicit_include_by_regex_policy"
+	DISKS_POLICY_EXCLUDE              = "explicit_exclude_policy"
+	DISKS_POLICY_EXCLUDE_BY_REGEX     = "explicit_exclude_by_regex_policy"
+	DISKS_POLICY_CONFLICTING          = "unallowed_definition_in_include_and_exclude_policy"
 	DISKS_POLICY_CONFLICTING_BY_REGEX = "unallowed_definition_in_include_or_exclude_and_contradicting_regexp"
-	DISKS_POLICY_NO_MATCH_FALLBACK = "not_matching_fallback_to_all_others"
+	DISKS_POLICY_NO_MATCH_FALLBACK    = "not_matching_fallback_to_all_others"
 )
```

Comment:
@thomashoell Please use CamelCase for the consts. You have used that format in backup/definitions.go also.

Reply:
Fixed in https://github.com/dreitier/backmon/commit/6ba9e3014a4877811bc1d5a2fd10bfe0c046de91